vm/src/builtins.rs

```diff
@@ -613,6 +613,24 @@ impl Printer for std::io::StdoutLock<'_> {
     }
 }
 
+pub fn builtin_exit(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(vm, args, required = [], optional = [(object, None)]);
+    match object {
+        Some(value) => match i32::try_from_object(&vm, value.clone()) {
+            Ok(code) => std::process::exit(code),
+            _ => {
+                let stdout = io::stdout();
+                let mut printer: Box<dyn Printer> = Box::new(stdout.lock());
+                printer.write(vm, value.clone())?;
+                printer.write(vm, "\n".into_pyobject(vm).unwrap())?;
+                printer.flush(vm)?;
```

Comment:
Instead of all this, you can just do:

```rust
println!("{}", vm.to_str(&obj)?.as_str());
```

Reply:
Thanks !
I made a new commit with the suggested changes.
I noticed that std::process::exit won't work in the wasm build for obvious reasons. Any suggestions on what could be done in this case ?

Reply:
It looks like the test is passing for wasm, or do you mean that you tried it out, and it doesn't work? If that's the case, I'd say use `panic!()` to exit the program.

Reply:
Indeed, panic! will cause the program to exit, but will terminate with code 101 and std::process::exit() will cause a panic in the wasm built anyway.
I also noticed that the exit function won't print anything in the wasm build, what could I use instead of println! in this case ?

Reply:
> std::process::exit() will cause a panic in the wasm build anyway (maybe this is ok ?)

It is ok; `exit` as a function doesn't really exist in wasm because when a wasm program aborts, it still doesn't quit the entire web page or anything like that.

> I also noticed that the builtin_exit function won't print anything in the wasm build, what could I use instead of println! in this case ?

I think it's fine for now, it doesn't really matter for now. Python in the browser isn't well defined, so it's fine if it doesn't work exactly how it would on the command line.

Reply:
Thanks for the clarification.